src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -97,8 +104,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
         }
 
-        data.comment = feedbackResponseComment;
-
+        // createdAt, lastEditedAt, commentGiverName and lastEditorName are required to generate editedCommentDetails
+        data.comment = updatedComment;
+        data.question = logic.getFeedbackQuestion(response.feedbackQuestionId);
+        data.showCommentToString = StringHelper.toString(updatedComment.showCommentTo, ",");
+        data.showGiverNameToString = StringHelper.toString(updatedComment.showGiverNameTo, ",");
+        data.instructorEmailNameTable = instructorEmailNameTable;
```

Comment:
Looks like `question`, `showCommentToString`, `showGiverNameToString` and `instructorEmailNameTable` are not needed at all to generate `editedCommentDetails` - they seem to be there only to create the `FeedbackResponseCommentRow`.

Instead of constructing an entire `FeedbackResponseCommentRow` when we're not using it at all, a better approach would be to pass the giver name and editor name to `createEditedCommentDetails` from here. This way, the `createEditedCommentDetails` method would depend only on the `comment.createdAt` and `comment.lastEditedAt` fields and the `sessionTimeZone`. This would also make the code much clearer exactly which fields are being used, without having to examine `getComment` and the `FeedbackResponseCommentRow` constructor to see which fields are actually being used. It will also help save one datastore call (`logic.getFeedbackQuestion` is unnecessary).